lib/src/gnav.dart

```diff
@@ -33,7 +33,7 @@ class GNav extends StatefulWidget {
 
   final List<GButton> tabs;
   final int selectedIndex;
-  final Function onTabChange;
+  final Function(int) onTabChange;
```

Comment:
Could you change the type for ValueChanged<int>?

Reply:
Done, good catch, thanks